src/platforms/react-native/manual-setup/hermes.mdx

```diff
@@ -16,19 +16,49 @@ For Sentry open source, self-hosted users, the minimum version required is [f073
 
 Other than making sure you have the minimum version of the SDK, Sentry itself is all you need besides the standard integration described in the [React Native Sentry documentation](/clients/react-native/).
 
-## Custom SourceMaps
+## Source Maps
 
-The combination of `@sentry/react-native` and `@sentry/cli` should integrate
-correctly into the `react-native` build pipeline to find and upload the correct source map.
+If you do not need custom sourcemaps, the `sentry.gradle` build step fully supports hermes sourcemaps and should handle everything for you.
+
+## Custom Source Maps
 
 However, care must be taken to upload the correct source map when manually bundling and building React Native apps.
 
 Building Hermes bundles is a three-step process, each of which creates a different
 source map.
 
-1. Bundling / minifying using `metro`: `index.android.bundle.packager.map`
-2. Compiling to bytecode using `hermes`: `index.android.bundle.compiler.map`
-3. Merging SourceMaps using `compose-source-maps`: `index.android.bundle.packager.map + index.android.bundle.compiler.map => index.android.bundle.map`
+1. Bundling / minifying using `metro` (`react-native`): `.packager.map`
+2. Compiling to bytecode using `hermes`: `.compiler.map`
+3. Merging SourceMaps using `compose-source-maps`: `.packager.map + .compiler.map => .map`
+
+```bash{tabTitle:iOS}
+$ react-native bundle --platform ios --entry-file index.js --bundle-output main.jsbundle --sourcemap-output main.jsbundle.packager.map
+$ hermes -O -emit-binary -output-source-map -out=main.jsbundle.compiler.map main.jsbundle.packager.map
+$ node react-native/scripts/compose-source-maps.js main.jsbundle.packager.map main.jsbundle.compiler.map -o main.jsbundle.map
```

Comment:
do we actually need the `$`? I don't see this for another bash scripts

Reply:
Hmm I kept it from the original comment from @Swatinem for ease of reading to make it clear that these are separate commands due to how long they are, I will add line breaks between them then to keep consistent with other code blocks.